Fundamentos de la Programacion/fundamentos_programacion.md

```diff
@@ -116,6 +116,45 @@ Para terminar, en la última ventana tenemos también opciones para copiar el pr
 - [Ficha docente de la asignatura Fundamentos de la Programación del curso 2016/2017](http://www.fdi.ucm.es/Pub/ImpresoFichaDocente.aspx?Id=913)
 - [Eclipse](https://eclipse.org/)
  - [Eclipse C/C++](http://www.eclipse.org/downloads/packages/eclipse-ide-cc-developers/neon2)
+ 
+ 
+ ## Atom Editor 
+### 1. Descripción del programa
+
+Atom es un editor de texto de código abierto desarrollado por Github usando Electron, que permite crear aplicaciones multiplataforma usando tecnologías web basadas en Node.js
+### 2. Requisitos hardware
+Cualquier versión reciente de GNU/Linux, MacOs o Windows
+### 3. Guía de instalación
+#### Debian y derivadas
+Para la instalación necesitamos descargar el .deb de [aquí](https://atom.io) y posteriormente abrir una terminal teniendo acceso al usuario administrador para poder ejecutar el siguiente comando:
```

Comment:
Dado que Atom no soporta la actualización automática ni a través de la aplicación, [ni a través de repositorios (oficiales)](https://github.com/atom/atom/issues/2956) para las distintas distribuciones Linux; sería interesante añadir la opciones de instalación a través de los contenedores/paquetes snap ([hay una oficial](https://insights.ubuntu.com/2017/05/11/atom-is-now-available-as-a-snap-for-ubuntu/)) y por flatpack (aunque sin versión oficial, ver: https://github.com/atom/atom/issues/11837 y https://github.com/atom/atom/pull/13293)

Reply:
Hecho, aunque también hay un [PPA no oficial](https://launchpad.net/~webupd8team/+archive/ubuntu/atom)